stdlib/public/Platform/Platform.swift

```diff
@@ -381,17 +438,25 @@ internal var _ignore = _UnsupportedPlatformError()
 // semaphore.h
 //===----------------------------------------------------------------------===//
 
+#if !os(Windows) || CYGWIN
 /// The value returned by `sem_open()` in the case of failure.
 public var SEM_FAILED: UnsafeMutablePointer<sem_t>? {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
   // The value is ABI.  Value verified to be correct for OS X, iOS, watchOS, tvOS.
   return UnsafeMutablePointer<sem_t>(bitPattern: -1)
-#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Windows)
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android)
+  // The value is ABI.  Value verified to be correct on Glibc.
+  return UnsafeMutablePointer<sem_t>(bitPattern: 0)
+#elseif os(Windows)
```

Comment:
Its behind a `!os(Windows) || CYGWIN`.  The `CYGWIN` will report as `os(Windows)`.  There is a thread on swift-dev for this, Ill send a proposal to swift-evolution for improving this.  This is just a means for getting stuff working for now.  Yes, its confusing, and not particularly elegant :-(.